templates/registration/login.html

```diff
@@ -0,0 +1,16 @@
+{%  extends "base.html" %}
+
+{% block content %}
+    <h1>Login</h1>
+
+    {% if form.errors %}
+        <p style="color: red">Invalid credentials</p>
+    {% endif %}
+
+    <form method="post" action="{% url 'login' %}">
+        {% csrf_token %}
+        {{ form.as_p }}
+
+        <input type="submit" value="login">

```

Comment:
it would be good to add hidden input so that user was redirected to the page they wanted to see after login